config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
```

Comment:
Please ensure that your `user#index` route is similar to the image below according to the project requirement.
Your Implementation | routes
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/167884796-a1056fd5-6fba-45a8-91fa-275ee2794494.png) | ![image](https://user-images.githubusercontent.com/10332499/167884905-40cded86-e11c-4978-8bdd-69979c290e73.png)